### PR TITLE
WFP-1880 merged peterborough and wisbech

### PR DESCRIPTION
**Pull request by carlov20**

### CODE REVIEW

File: src/main/resources/db/migration/V1_8__MERGE_PETERBOROUGH_WISBECH.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+update team set name = 'CBS-Peterborough & Wisbech SM' where code = 'N56332';
+delete from team where code = 'N56BAR';
```
Comment (carlov20) [inline src/main/resources/db/migration/V1_8__MERGE_PETERBOROUGH_WISBECH.sql:2]: introduce a soft delete instead?

Comment (sp-watson) [reply src/main/resources/db/migration/V1_8__MERGE_PETERBOROUGH_WISBECH.sql:2]: Not sure whether we need a soft delete - that depends on whether and where we store a reference to this.

But I think the delete has to happen some time after the add - if someone is doing an allocation using that team will it not fail?

Comment (carlov20) [reply src/main/resources/db/migration/V1_8__MERGE_PETERBOROUGH_WISBECH.sql:2]: we store reference to the team code in workload:
- https://github.com/ministryofjustice/hmpps-workload/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/entity/EventManagerEntity.kt#L34  
- https://github.com/ministryofjustice/hmpps-workload/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/entity/PersonManagerEntity.kt#L34 
- https://github.com/ministryofjustice/hmpps-workload/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/entity/RequirementManagerEntity.kt#L38 

and in allocations:
- https://github.com/ministryofjustice/hmpps-allocations/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/jpa/entity/UnallocatedCaseEntity.kt#L32

Comment (carlov20) [reply src/main/resources/db/migration/V1_8__MERGE_PETERBOROUGH_WISBECH.sql:2]: not sure there is an agreed place for where all historic probation estate data will reside or whether this service will be a currently active store

**Pull request closed without merging**